lib/Runtime/Library/JavascriptString.cpp

```diff
@@ -2134,37 +2134,51 @@ namespace Js
     Var JavascriptString::ToCaseCore(JavascriptString* pThis, ToCase toCase)
     {
         charcount_t count = pThis->GetLength();
+        charcount_t countToCase = count;
```

Comment:
`countToCase` isn't used between here and the next assignment to it. This is a dead assignment and you can remove it and move the definition down below where it is assigned next.